presto-hive/src/main/java/io/prestosql/plugin/hive/BackgroundHiveSplitLoader.java

```diff
@@ -227,6 +236,16 @@ public TaskStatus process()
                     return TaskStatus.finished();
                 }
                 ListenableFuture<?> future;
+                /*
```

Comment:
nit: I think in methods we use:
```
// comment
//  second line
```
style of comments.

Reply:
changed the style